app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AddProductOverviewFragment.java

```diff
@@ -793,7 +793,8 @@ void toastEmbCodeHint() {
     void searchProductLink() {
         String url = "https://www.google.com/search?q=" + code;
         if (!brand.getText().toString().isEmpty()) {
```

Comment:
The condition could also be changed to `!brand.getChipValues().isEmpty()`

Reply:
This looks like a corner case however I don't think it would cause any issue so we can consider it as the intended behavior.

Reply:
Okay, done. I've used `brand.getChipAndTokenValues()` instead so both chipped and non chipped brand names are in the query.